.github/dependabot.yml

```diff
@@ -4,3 +4,11 @@ updates:
     directory: "/"
     schedule:
       interval: "daily"
+  - package-ecosystem: "docker"
+    directory: "helm-charts/splunk-otel-collector"
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "docker"
+    directory: "helm-charts/splunk-otel-collector"
+    schedule:
+      interval: "daily"
```

Comment:
Did you add this twice? I don't see a difference.

Reply:
Whoops, it was supposed to be the examples dir. Fixing....